### PR TITLE
test: add connector-mock-sms and connector-mock-email for integration tests

### PR DESCRIPTION
**Pull request by IceHe**
## Summary

- Add `connector-mock-sms` and `connector-mock-email` for integration tests
- Replace `connector-twilio-sms` and `connector-sendgrid-mail` with `connector-mock-sms` and `connector-mock-email` 

## Testing

Ditto

### CODE REVIEW

File: .github/workflows/integration-test.yml [hunk 1]
```diff
@@ -93,6 +93,7 @@ jobs:
           INTEGRATION_TEST: true
           NODE_ENV: production
           DB_URL_DEFAULT: postgres://postgres:postgres@localhost:5432
+          ADDITIONAL_CONNECTOR_PACKAGES: '@logto/connector-mock-sms,@logto/connector-mock-email'
```
Comment (wangsijie) [inline .github/workflows/integration-test.yml:96]: nit: `'` is not necessary

**Review by wangsijie — APPROVED** [review]
LGTM

File: .github/workflows/integration-test.yml (hunk 1 above)
Comment (IceHe) [reply .github/workflows/integration-test.yml:96]: No. Since `@` is a reserved character in yaml, if we do not add `'`, it will fail.

**Comment by IceHe** [comment]
Rebased master to resolve conflicts.

**Pull request merged**